src/components/App.tsx

```diff
@@ -5658,9 +5657,12 @@ class App extends React.Component<AppProps, AppState> {
 
       let newZoom = this.state.zoom.value - delta / 100;
       // increase zoom steps the more zoomed-in we are (applies to >100% only)
-      newZoom += Math.log10(Math.max(1, this.state.zoom.value)) * -sign;
-      // round to nearest step
-      newZoom = Math.round(newZoom * ZOOM_STEP * 100) / (ZOOM_STEP * 100);
+      // reduced amplification for small deltas (small movements on a trackpad)
+      const smallDeltaAdjustment = Math.min(1, absDelta / 20);
+      newZoom +=
+        Math.log10(Math.max(1, this.state.zoom.value)) *
+        -sign *
+        smallDeltaAdjustment;
```

Comment:
```suggestion     
      newZoom +=
        Math.log10(Math.max(1, this.state.zoom.value)) *
        -sign *
         // reduced amplification for small deltas (small movements on a trackpad)
         Math.min(1, absDelta / 20)
```

I think it's better to inline this.

Btw, doesn't that expression _amplifies_ (clamped to 1) instead of reducing the amplification?

Reply:
It's going to result in multiplying by a number between 0 and 1, so it will _reduce_ the amplification.

Reply:
nvm. Even though I myself said we're clamping to `1`, my mind was for some reason in >1 numbers :).